src/knx/rf_physical_layer.h

```diff
@@ -182,6 +182,9 @@ extern void delayMicroseconds (unsigned int howLong);
 #define TX_ACTIVE 4
 #define TX_END 5
 
+// Calculate the real packet size out of the L-field of FT3 frame data. See KNX-RF spec. 3.2.5 Data Link Layer frame format
+#define PACKET_SIZE(lField) ((((lField - 10 /*size of first pkt*/))/16 + 2 /*CRC in first pkt */) * 2 /*to bytes*/ +lField + 1 /*size of len byte*/)
```

Comment:
I have written some testcode which runs this MACRO against the other packetSize() implementation for all input values from 0 to 255. Both calculate the same packet size for each input value.
So I replaced the old implementeation packetSize() with the MACRO.